app/src/organisms/ChangePipette/__tests__/CheckPipettesButton.test.tsx

```diff
@@ -47,29 +42,19 @@ describe('CheckPipettesButton', () => {
     props = {
       robotName: 'otie',
       children: <div>btn text</div>,
-      hidden: false,
       onDone: jest.fn(),
     }
     dispatchApiRequest = jest.fn()
-    mockFeatureFlag.mockReturnValue(true)
     mockGetRequestById.mockReturnValue(null)
     mockUseDispatchApiRequests.mockReturnValue([dispatchApiRequest, ['id']])
   })
   afterEach(() => {
     jest.resetAllMocks()
   })
 
-  it('renders the button without ff and clicking on it calls fetchPipettes', () => {
-    mockFeatureFlag.mockReturnValue(false)
-    const { getByRole } = render(props)
-    const btn = getByRole('button', { name: 'btn text' })
-    fireEvent.click(btn)
-    expect(dispatchApiRequest).toBeCalledWith(mockFetchPipettes('otie'))
-  })
-  it('renders the confirm attachment btn with ff and clicking on it calls fetchPipettes', () => {
+  it('renders the confirm attachment btn  and clicking on it calls fetchPipettes', () => {
```

Comment:
seems that after btn there is an extra space in the test description. 
```suggestion
  it('renders the confirm attachment btn and clicking on it calls fetchPipettes', () => {
```